packages/components/src/Dropdown/Dropdown.tsx

```diff
@@ -165,6 +189,68 @@ function childrenHaveLabelComponent(children?: ReactNode): boolean {
  *   <Dropdown aria-label="Options:" buttonText="Select" options={options} />
  * );
  * ```
+ *
+ * For compact dropdown button, add compact and optionsWidth props to
+ * <Dropdown>.
+ *
+ * Examples:
+ *
+ *
+ * ```
+ * return (
+ *   <Dropdown
+ *     aria-label="Options"
+ *     compact
+ *     optionsAlign="right"
+ *     optionsWidth="w-96"
+ *   >
+ *     <Dropdown.Options>
+ *       <Dropdown.Option>Option 1</Dropdown.Option>
+ *       <Dropdown.Option>Option 2</Dropdown.Option>
+ *       <Dropdown.Option>Option 3</Dropdown.Option>
+ *     </Dropdown.Options>
+ *   </Dropdown>
+ * );
+ * ```
+ *
+ * ```
+ * const options = [
+ *   ...
+ * ];
+ *
+ * return (
+ *   <Dropdown
+ *     aria-label="Options"
+ *     compact
+ *     options={options}
+ *     optionsAlign="right"
+ *     optionsWidth="w-96"
+ *   />
+ * );
+ * ```
+ *
+ * For dropdown that differs in button and options menu width, style <Dropdown>
+ * with className for the button with and provide optionsWidth for the options
```

Comment:
Should this say "button width"?
```suggestion
 * with className for the button width and provide optionsWidth for the options
```